metro.config.windows.js

```diff
@@ -6,13 +6,12 @@
  */
 const fs = require('fs');
 const path = require('path');
-const blacklist = require('metro-config/src/defaults/blacklist');
 
 const rnPath = fs.realpathSync(
```

Comment:
shouldn't this file go into the example folder?

Reply:
There was a note on the RNW website that mentioned that metro.config.windows.js should be in the root of the repo. "Add metro.config.windows for Windows bundling (example). Make sure the config file is at the root of the repo (see Metro bug #588)." I can test to see if the code still works when the files are moved?